WordPress/Learn
# Plugin: Disable functions that add links/references to GitHub, plus other cleanup

This started as a simple fix for #85, removing the filters that change post edit links to go to GitHub, since markdown import is now disabled. However, that kind of snowballed into a bunch of other small cleanup and reorg commits.

* Remove the functions that modify edit links.
* Remove the entire `Lesson_Plan` class, since everything in it has now been either relocated or removed entirely.
* Remove the block of CSS that was being injected into every front end page load.
* Move the filter modifying the excerpt length over to the theme.
* Reorganize action/filter additions so they are in the same files where the corresponding functions live.
* Add a namespace to the root plugin file.
* Remove all the shortcode functionality, as it was already commented out and apparently never used.

Fixes #85 

### To Test

Generally make sure nothing is broken.

> Remove the block of CSS that was being injected into every front end page load.

This change means we need to do a visual audit of the front end of the site, in case any theme CSS we have was unknowningly assuming that the injected style block was present.
